### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mohankumar80**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,36 +9,86 @@
...
                 <a href="#inputs" class="side-item">Input</a>
                 <a href="#lists" class="side-item">List</a>
-                <a href="#navigation" class="side-item">Navigation</a>
                 <a href="#modals" class="side-item">Modal</a>
+                <a href="#navigation" class="side-item">Navigation</a>
                 <a href="#ratings" class="side-item">Rating</a>
                 <a href="#sliders" class="side-item">Slider</a>
-                <a href="#grids" class="side-item">Grid</a>
                 <a href="#utilities" class="side-item">Text-Utilities</a>
                 <a href="#toasts" class="side-item">Toast</a>
```
Comment (DanishOnGit) [inline index.html]: Hi Mohan,
Can we use a list tag here? Because that would be semantic I guess.

File: one.css [hunk 2]
```diff
@@ -417,20 +432,145 @@ Modal
...
 }
 
-/* Grid with 3 colums */
-.grid-col-3 {
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr;
+.navbar-nav {
+    padding: 0;
+    margin: 0;
+}
```
Comment (DanishOnGit) [inline one.css:485]: Can we do the following? (assuming u want to remove default margins and paddings from all elements)
```
*{
margin: 0;
padding:0;
}
```
Please let me know if this doesn't work..

File: styles.css [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
+/* Side nav */
+aside {
+    grid-area: sidebar;
+    position: fixed;
+    top: 5rem;
 }
 
 .side-item {
     border-radius: 0.2rem;
     color: #374151;
```
Comment (DanishOnGit) [inline styles.css]: u can add a color var here too

File: styles.css [hunk 4]
```diff
@@ -64,10 +89,54 @@ main {
...
+    border-radius: 1rem;
+    color: white;
+    display: inline-block;
+    font-size: 1rem;
+    padding: 1rem;
+    width: 90%;
+}
+
+.utilites > h3 {
+    color: #FF7518;
```
Comment (DanishOnGit) [inline styles.css]: color variables here too

**Review by DanishOnGit — COMMENTED** [review]
Hey Mohan,
I suggest you divide your html markup using section and article tags for better semantic and code readability. Other than this,  I have simply pointed out places, where you can add some more css variables.

File: index.html (hunk 1 above)
Comment (mohankumar80) [reply index.html]: done

File: styles.css (hunk 3 above)
Comment (mohankumar80) [reply styles.css]: done

File: styles.css (hunk 4 above)
Comment (mohankumar80) [reply styles.css]: done

File: one.css (hunk 2 above)
Comment (mohankumar80) [reply one.css:485]: We can do the following you mentioned and its working. Its removing padding and margin on every element of the page which I don't need.What I'm trying to do is removing the padding and margin on the logo of the website and the list that contains the items on the navigation bar.

File: one.css [hunk 5]
```diff
@@ -225,6 +225,9 @@ Buttons
     border-radius: 50%;
     background-color: var(--orange);
     font-size: 1rem;
+    width: 2.5rem;
+    height: 2.5rem;
+    outline: none;
```
Comment (neharajesh) [inline one.css]: Hey! Apparently outline: none should not be used. Maybe try outline-color: transparent? Hope this helps!

File: one.css [hunk 6]
```diff
@@ -225,6 +225,9 @@ Buttons
...
 }
 
 .btn-float:hover {
```
Comment (neharajesh) [inline one.css:239]: maybe you can also change cursor to pointer on hover!

**Review by neharajesh — COMMENTED** [review]
Hey Mohan! 
Well written code! I've pointed out a few additions/changes that might be useful, please do check them out!

File: one.css (hunk 5 above)
Comment (mohankumar80) [reply one.css]: done

File: one.css (hunk 6 above)
Comment (mohankumar80) [reply one.css:239]: done

**Comment by mohankumar80** [comment]
> Hey Mohan,
> I suggest you divide your html markup using section and article tags for better semantic and code readability. Other than this, I have simply pointed out places, where you can add some more css variables.

**Review by gkanishk — COMMENTED** [review]
@mohankumar80 here are few suggestions:
- Link buttons should have `text-decoration: underline` on hover not by default.
  - Screenshot 2021-03-29 at 8 28 20 PM

- Rounded image doesn't appear to be rounded, you can make three different images of different sizes and define equal dimensions for width and height.
  - Screenshot 2021-03-29 at 8 28 31 PM


- Close button used inside the card and toast try to use less vertical padding elongated button looks bit odd.
  - Screenshot 2021-03-29 at 8 28 45 PM

**Comment by mohankumar80** [comment]
Done

**Pull request merged**